python/py-tokenizers/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tokenizers
+version             0.9.0
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         Fast and Customizable Tokenizers
+long_description    🤗 Tokenizers provides an implementation of today's \
+                    most used tokenizers, with a focus on performance \
+                    and versatility.
+
+homepage            https://github.com/huggingface/tokenizers
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    checksums       ${distname}${extract.suffix} \
+                    rmd160  103bd238583abb0abb2a867ea59eef2c134e7c75 \
+                    sha256  0bfa960345e114efd553e265e32eca6d79861abe24925fba903c925f8760f795 \
+                    size    170434
```

Comment:
please move the checksums outside of the `if`-block; also `${distname}${extract.suffix}` shouldn't be needed I think

Reply:
Thanks done.